scripts/twister

```diff
@@ -772,6 +772,12 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
              "stdout detailing RAM/ROM sizes on the specified filenames. "
              "All other command line arguments ignored.")
 
+    parser.add_argument("--serial-decode-specified", action="store_true",
```

Comment:
@enjiamai , please add this option in alpha-beta order by the leding character, in this case it is 's'

Reply:
Hi @hakehuang , I think I won't use an extra args/options in the coming update. But you are right, it should follow this alpha-beta order if an option need to be added. Thanks for the reminder!